code/datums/flock/flock.dm

```diff
@@ -40,6 +38,8 @@
 	processing_items |= src
 	for(var/DT in childrentypesof(/datum/unlockable_flock_structure))
 		src.unlockableStructures += new DT(src)
+	if (!annotation_imgs)
```

Comment:
Not necessary considering New() should only be called once?

Reply:
`annotation_imgs` is static, so it's shared between all instances